Release 4.1.4 (#58)
blackbaud/skyux-avatar: CHANGELOG.md

# 4.1.4 (2021-07-13)

> - Fixed the avatar component to allow for loading files with parenthesis in the file name. [#57](https://github.com/blackbaud/skyux-avatar/pull/57)

Reviewer: ```suggestion
- Fixed the avatar component to let users load files with a parenthesis in the filename. [#57](https://github.com/blackbaud/skyux-avatar/pull/57)
```
